tasks/gulp/clean.js

```diff
@@ -26,7 +26,7 @@ src.displayName = 'clean:src'
 src.description = 'Cleans `govuk` folder from src'
 
 const pkg = async (done) => {
-  await cleanupFoldersOrFiles(['package/govuk', 'package/govuk-frontend', 'package/digitalmarketplace'])
+  await cleanupFoldersOrFiles('package/digitalmarketplace')
```

Comment:
Can we have this still delete `package/govuk` to avoid devs like me making silly mistakes? 😁

Reply:
We can and we should!